.travis.yml

```diff
@@ -29,26 +29,10 @@ scala:
   - 2.12.2
 
 env:
-  - TEST_PACKAGE=core-macros/test
-  - TEST_PACKAGE=core-sharedJVM/test
-  - TEST_PACKAGE=core-sharedJS/test
-  - TEST_PACKAGE=core-frontend/test
-  - TEST_PACKAGE=rpc-macros/test
-  - TEST_PACKAGE=rpc-sharedJVM/test
-  - TEST_PACKAGE=rpc-sharedJS/test
-  - TEST_PACKAGE=rpc-backend/test
-  - TEST_PACKAGE=rpc-frontend/test
-  - TEST_PACKAGE=rest-macros/test
-  - TEST_PACKAGE=rest-sharedJVM/test
-  - TEST_PACKAGE=rest-sharedJS/test
-  - TEST_PACKAGE=i18n-sharedJVM/test
-  - TEST_PACKAGE=i18n-sharedJS/test
-  - TEST_PACKAGE=i18n-backend/test
-  - TEST_PACKAGE=i18n-frontend/test
+  - TEST_PACKAGE="core-macros/test core-sharedJVM/test core-sharedJS/test core-frontend/test"
```

Comment:
Tests usually fail because of Selenium/Chrome crash. Now it spends less time on compiling project and it has two chances to fail (sharedJS and frontend). I think it will be faster now.